ChaosMod/Effects/EffectsInfo.h

```diff
@@ -559,4 +560,5 @@ const std::unordered_map<EffectType, EffectInfo> g_effectsMap =
 	{EFFECT_MISC_FPS_LIMIT, {"Console Experience", "misc_fps_limit", true, {}, true}},
 	{EFFECT_META_NO_CHAOS, {"No Chaos", "meta_nochaos", true, { EFFECT_META_HIDE_CHAOS_UI }, false, EffectExecutionType::META}},
 	{EFFECT_PEDS_ROASTING, {"Roasting", "peds_roasting", true, {}, true}}
+	{EFFECT_PLAYER_AIMBOT, {"Aimbot", "player_aimbot", true}},
```

Comment:
Doesn't compile as is, the trailing comma should be on the previous line. Same thing in the ConfigApp/Effects.cs file.

Reply:
thats embarassing 😅